bindings/azure/blobstorage/metadata.yaml

```diff
@@ -0,0 +1,103 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: azure.blobstorage
+version: v1
+status: stable
+title: "Azure Blob Storage"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/blobstorage/
+binding:
+  output: true
+  operations:
+    - name: create
+      description: "Create blob."
+    - name: get
+      description: "Get blob."
+    - name: delete
+      description: "Delete blob."
+    - name: list
+      description: "List blob."
+capabilities: []
+authenticationProfiles:
+  - title: "Connection string"
+    description: "Authenticate using a connection string."
+    metadata:
+      - name: connectionString
+        required: true
+        sensitive: true
+        description: "The Service Bus connection string."
+        example: '"BlobEndpoint=https://storagesample.blob.core.windows.net;SharedAccessSignature={KeySig}"'
+        # If omitted, uses the same values as "<root>.binding"
+        binding:
+          output: true
+  - title: "Account Key"
+    description: |
+      Authenticate using a pre-shared "account key".
+    metadata:
+      - name: accountKey
+        required: true
+        sensitive: true
+        description: "The key to authenticate to the Storage Account or Cosmos DB Table API."
+        example: '"my-secret-key"'
+builtinAuthenticationProfiles:
+  - name: "azuread"
+metadata:
+  - name: accountName
```

Comment:
I believe this is required if using "azuread" and the "Account Key" profile, but looks like the connection string profile already includes this?